source/core/Synchronization.cpp

```diff
@@ -765,7 +765,7 @@ VIREO_FUNCTION_SIGNATURE6(QueueRef_Obtain, RefNumValType*, Int32, StringRef, Boo
         if (createdPtr)
             *createdPtr = false;
         if (errPtr)
-            errPtr->SetError(true, errCode, "ObtainQueue");
+            errPtr->SetError(true, errCode, "ObtainQueue", true);
```

Comment:
I don't understand the question re: something being handled in the SetError function.  Format Into String needs to add "(arg N)" between the prim name and the call chain, so it has to be slightly different.  But most prims can use SetError.